app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -37,33 +37,42 @@ export class ActiveOrders extends React.Component {
       uuid,
       drug,
       dose,
-      dosingUnit,
+      doseUnits,
       frequency,
       route,
       duration,
-      durationUnit,
-      reason,
-      drugInstructions,
-      dispensingQuantity,
-      dispensingUnit,
+      careSetting,
+      durationUnits,
+      asNeededCondition,
+      orderer,
+      dosingInstructions,
+      dosingType,
+      type,
+      quantity,
+      quantityUnits,
       orderNumber,
     } = order;
 
     this.setState({
       draftOrder: {
         uuid,
+        careSetting: careSetting.uuid,
         drugName: drug.display,
         action: 'DISCONTINUE',
         dose,
-        dosingUnit,
-        frequency,
-        route,
+        drug: drug.uuid,
+        dosingUnit: (doseUnits && doseUnits.display) || doseUnits,
```

Comment:
Hey. What is this line suppose to do? Just curious.

Reply:
it checks that there is a valid "value" in "doseUnits", if so then return "doseUnits.display" or else return whatever value is in "doseUnits", which can either be "null" or empty string " ".